Jenkinsfile

```diff
@@ -88,51 +108,58 @@ def publishChart(imageName) {
 }
 node {
   checkout scm
-  stage('Set branch, PR, and containerTag variables') {
-    (branch, pr, containerTag, mergedPrNo) = getVariables(repoName)
-    if (pr ) {
-      sh "echo Building $pr"
-    } else if (branch == "master") {
-      sh "echo Building master branch"
-    } else {
-      currentBuild.result = 'ABORTED'
-      error('Build aborted - not a PR or a master branch')
+  try {
+    stage('Set branch, PR, and containerTag variables') {
+      (branch, pr, containerTag, mergedPrNo, repoUrl, commitSha) = getVariables(repoName)
+      if (pr ) {
+        sh "echo Building $pr"
+      } else if (branch == "master") {
+        sh "echo Building master branch"
+      } else {
+        currentBuild.result = 'ABORTED'
+        error('Build aborted - not a PR or a master branch')
+      }
+      updateGithubCommitStatus('Build started','PENDING',repoUrl, commitSha)
```

Comment:
Being massively nitpicky now, but inconsistent spacing on the , in the method parameters.